wazo_confd/sync_db.py

```diff
@@ -102,10 +102,11 @@ def main():
 
 
 def remove_tenant(tenant_uuid):
-    for user in user_dao.find_all_by(tenant_uuid=tenant_uuid):
-        logger.debug('Removing user: %s', user.uuid)
-        user_dao.delete(user)
+    logger.debug('Removing tenant: %s', tenant_uuid)
+    with session_scope():
+        tenant = tenant_resources_dao.get(tenant_uuid)
+        tenant_resources_dao.delete(tenant)
 
-    # FIXME(fblackburn):
-    # * Add all other resources related to tenant_uuid
-    # * Reset device to autoprov
+
```

Comment:
This is not required, the `console_scripts` in setup.py should automatically add a launcher

Reply:
perfect, but is it a problem if I let that ? (to be able call manually the python script)